File: BBIS.Application/DTOs/ApplicationSetting/BloodComponentSettingDto.cs
namespace BBIS.Application.DTOs.ApplicationSetting
{
    public class BloodComponentSettingDto
    {
        public Guid? BloodComponentId { get; set; }
        public string ComponentName { get; set; }
        public int ExpiryInDays { get; set; }
        public int NotifyOnDaysBefore { get; set; }
        public bool IsActive { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/ApplicationSetting/BloodComponentSettingDto.ts
export interface IBloodComponentSettingDto {
    BloodComponentId: Guid;
    ComponentName: string;
    ExpiryInDays: number;
    NotifyOnDaysBefore: number;
    IsActive: boolean;
}

export class BloodComponentSettingDto implements IBloodComponentSettingDto {
    BloodComponentId: Guid = '';
    ComponentName: string = '';
    ExpiryInDays: number = 0;
    NotifyOnDaysBefore: number = 0;
    IsActive: boolean = true;
}